app/decorators/course_decorator.rb

```diff
@@ -91,7 +91,7 @@ def content_tag_content
     when 'draft'
       'Draft'
     when 'published_with_unpublished_changes'
-      'Published&nbsp;*'
+      'Published*'
```

Comment:
This is so we display `Published *` (with a space) and never `Published (new line) *`. No space should still be fine though.